test/openAPI/features/wallet_generate_link_auth_code.feature

```diff
@@ -48,10 +51,34 @@ Feature: The endpoint to validates the link-code and its expiry and generates th
+    And The link code, transaction and authenticate is completed before POST /linked-authorization/link-auth-code
     When Send POST /linked-authorization/link-auth-code request with given linkCode and transactionId and invalidRequestTime
     Then Receive a response from the /linked-authorization/link-auth-code endpoint
     And The /linked-authorization/link-auth-code endpoint response should be returned in a timely manner 15000ms
     And The /linked-authorization/link-auth-code endpoint response should have status 200
     And The /linked-authorization/link-auth-code endpoint response should have content-type: application/json header
     And The /linked-authorization/link-auth-code endpoint response should match json schema with errors
     And The /linked-authorization/link-auth-code response should contain errorCode property equals to "invalid_request"
+
+  @negative
+  Scenario: Not able to validate the link-code and its expiry because of transaction and link code are not connected to each other
+    Given Wants to validate the link-auth-code and generate the auth code
+    And The link code, transaction and authenticate is completed before POST /linked-authorization/link-auth-code
+    When Send POST /linked-authorization/link-auth-code request with given valid linkCode and transactionId
+    Then Receive a response from the /linked-authorization/link-auth-code endpoint
+    And The /linked-authorization/link-auth-code endpoint response should be returned in a timely manner 15000ms
+    And The /linked-authorization/link-auth-code endpoint response should have status 200
+    And The /linked-authorization/link-auth-code endpoint response should have content-type: application/json header
+    And The /linked-authorization/link-auth-code endpoint response should match json schema with errors
+    And The /linked-authorization/link-auth-code response should contain errorCode property equals to "invalid_link_code"
+
+  @negative
+  Scenario: Not able to validate the link-code and its expiry because of reuse of the completed transaction_id
+    Given Wants to validate the link-auth-code and generate the auth code
+    And The first authorization flow for transactionId ends
+    When Send POST /linked-authorization/link-auth-code request with given linkCode and reused completed transactionId
+    Then Receive a response for reuse transactionId from the /linked-authorization/link-auth-code endpoint
+    And The /linked-authorization/link-auth-code endpoint response for reuse transactionId should have status 200
+    And The /linked-authorization/link-auth-code endpoint response for reuse transactionId should have content-type: application/json header
+    And The /linked-authorization/link-auth-code endpoint response for reuse transactionId should match json schema with errors
+    And The /linked-authorization/link-auth-code response for reuse transactionId should contain errorCode property equals to "invalid_transaction_id"
```

Comment:
Moving comment from old PR (https://github.com/GovStackWorkingGroup/bb-identity/pull/34#discussion_r1218353480)

I give a similar comment on another PR and it was taken care, can you check this once ?

https://github.com/GovStackWorkingGroup/bb-identity/pull/32#discussion_r1210069340

Reply:
@KarolinaKopacz Can you help by pointing out which test case is taking care of this comment ?

Reply:
About https://github.com/GovStackWorkingGroup/bb-identity/pull/34#discussion_r1218353480
I tried to simulate it but it's more complicated than in another test case...
If the response will be after 25s then test will fail (without "response_timeout" error code because next test steps will not proceed). For example:
Step 1. Send a request - pass
Step 2. Check the response time - fail
Step 3 .... - also fail whatever it is
Step 4 .... - also fail whatever it is

So every time there is a response after 25s, the test will fail, and then the rest of the steps will not be executed

Reply:
ok, will try this again the future.